metadata-ingestion/tests/integration/cassandra/setup/cassandra.yaml

```diff
@@ -0,0 +1,1827 @@
+# ---------------------------------------------------------------------------------------- #
+# NOTE:
+# This yaml has been copied from the official Cassandra Docker image:
+# (https://github.com/datastax/docker-images/blob/master/config-templates/DSE/6.0.0/cassandra.yaml)
+# 
+# Key interests are in setting the AllowAllNetworkAuthorizer,
+# and in enabling materialized view feature.
```

Comment:
Other than these two, we don't explicitly need to set anything as there's already [defaults baked in](https://docs.datastax.com/en/cassandra-oss/3.0/cassandra/configuration/configCassandra_yaml.html#Quickstartproperties). Should I delete everything else from this file for the sake of keeping code minimal? Or is it good to keep around for reference? I'm leaning former.